src/main/java/Ui.java

```diff
@@ -0,0 +1,65 @@
+import java.util.Scanner;
+
+public class Ui {
+
+    private static final String LOGO =
+            " \n____        _        \n"
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private static final String DIVIDER =
+            "    ____________________________________________________________\n";
+
+//    private final Scanner sc;
+//
+//    Ui() {
+//        sc = new Scanner(System.in);
+//    }
+//
+//    String readCommand() {
+//        return sc.nextLine();
+//    }
```

Comment:
Perhaps it would make the code cleaner if these commented code are completely removed? Otherwise , leaving to the end of the java file is good too.